scripts/mkstatic.py

```diff
@@ -183,7 +183,7 @@ class API(object):
 {suite_switch}
       else
 
-         write({ccpp_var_name}%errmsg,'(*(a))'), 'Invalid suite ' // trim(suite_name)
+         write({ccpp_var_name}%errmsg,'(*(a))') 'Invalid suite ' // trim(suite_name)
```

Comment:
I do not understand the asterisk here. It appears to be something l like a 'general repeat count' but my reading of the standard says that repeat counts are always positive literal integers. On the other hand, Nag accepts this syntax, even for strict Fortran 2003 processing so I must be missing something. Do you have a format description of this syntax anywhere?

Reply:
No, I don't. I have seen the same as you, it works with all compilers I have tested and with N=1...MANY string variables.

Reply:
Right. Nag is the gold standard when it comes to being really picky about Fortran standards so the fact that it accepts this syntax means I am missing something. I have to go find a Fortran uber-expert . . .